src/batou_ext/oci.py

```diff
@@ -108,6 +172,12 @@ class Container(Component):
     }
 
     def configure(self):
+        self.backend = (
+            "podman"
+            if self.require("oci:podman", strict=False, host=self.host)
```

Comment:
Wondering whether we want to make it this implicit. 
This might cause a docker assuming container being executed by podman